package.json

```diff
@@ -45,6 +47,7 @@
     "https-browserify": "^1.0.0",
     "path-browserify": "^1.0.1",
     "process": "^0.11.10",
+    "raw-loader": "^4.0.2",
```

Comment:
I need this to overcome an issue related to the webpack trying to load an unknown *.txt file (the PT Mono license file) due to the added greedy context `@site/static/*` here `import('@site/static/${iconStaticFolderPath}')`
Another solution would be to delete the license file, but I'd rather keep it, just in case 😛